gotfparse/pkg/converter/converter.go

```diff
@@ -181,3 +271,85 @@ func (t *terraformConverter) SetDebug() {
 func (t *terraformConverter) SetStopOnHCLError() {
 	t.parserOptions = append(t.parserOptions, parser.OptionStopOnHCLError(true))
 }
+
+func (t *terraformConverter) visitModule(m *terraform.Module, out *gabs.Container) {
+	path := t.getModulePath(m)
+
+	for _, b := range m.GetBlocks() {
+		t.visitBlock(b, path, out)
+	}
+}
+
+func getModName(b *terraform.Block) string {
```

Comment:
```suggestion
func getModuleName(b *terraform.Block) string {
```

Reply:
Replace/rename `mod` with `getModuleName` and `moduleName` .. everything else uses `module` so I find myself having to pause for second when I encountered this.

Don't use this suggestion, there are a 4 or 5 places that need to be updated.